modules/apps/frontend-taglib/frontend-taglib-clay/package.json

```diff
@@ -10,7 +10,7 @@
 		"@clayui/color-picker": "3.2.4",
 		"@clayui/css": "3.13.0",
 		"@clayui/data-provider": "3.2.0",
-		"@clayui/date-picker": "3.3.0",
+		"@clayui/date-picker": "3.4.1",
```

Comment:
@linolaoj, have you discussed with anyone on Front-End Infrastructure team about this upgrade? I'm asking this because usually this team performs this upgrades.

Reply:
Hi @natocesarrego, I've opened https://issues.liferay.com/browse/IFI-1837 to have the Clay date picker fixed. That is why I'm upgrading the version. But I have not discussed it with someone from the Front-End team. I'll reach someone from the team to know if there is some additional procedure to be done.
Thanks!